doc/data/messages/t/too-complex/bad.py

```diff
@@ -0,0 +1,54 @@
+fruit = {'apple': 1.1,
+         'pear': 0.8,
+         'banana': 1.2,
+         'mango': 3.5,
+         'peach': 0.5,
+         'melon': 4.9,
+         'orange': 2.0,
+         'strawberry': 2.5,
+         'mandarin': 2.3,
+         'plum': 0.5,
+         'watermelon': 6.4
+         }
+
+discounted_fruit = ('banana', 'mango', 'orange', 'watermelon')
+shopping_list = []
+
+
+def fifty_percent_off(whole):
+    return (float(whole)) * 50 / 100
+
+
```

Comment:
It would be nice if the original function that is too big is named something like ``calculate_sum_and_display_price_of_fruits(fruits, discounted_fruits)``